<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmRadioamadores</class>
 <widget class="QDialog" name="frmRadioamadores">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>882</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imgs/imagens.qrc">
    <normaloff>:/radioamadores.png</normaloff>:/radioamadores.png</iconset>
  </property>
  <widget class="QPushButton" name="btnAdicionarComentario">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>390</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string extracomment="Adicionar Comentário"/>
   </property>
   <property name="toolTipDuration">
    <number>2</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="imgs/imagens.qrc">
     <normaloff>:/mais.png</normaloff>:/mais.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnRemoverComentarios">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>390</y>
     <width>30</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>9</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="toolTip">
    <string extracomment="Remover Comentário"/>
   </property>
   <property name="toolTipDuration">
    <number>2</number>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="imgs/imagens.qrc">
     <normaloff>:/menos.png</normaloff>:/menos.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>20</width>
     <height>20</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>190</y>
     <width>251</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>QRA</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtQra"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>190</y>
     <width>231</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>Nome</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtNome"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>190</y>
     <width>361</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Sobrenome</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtSobrenome"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>491</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Cidade</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtCidade"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>250</y>
     <width>211</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_8">
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Estado</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtEstado"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>250</y>
     <width>141</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_9">
    <item>
     <widget class="QLabel" name="label_10">
      <property name="text">
       <string>País</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtPais"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>310</y>
     <width>211</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_10">
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>Celular</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtCelular"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>491</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_11">
    <item>
     <widget class="QLabel" name="label_12">
      <property name="text">
       <string>E-Mail</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtEmail"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>430</y>
     <width>861</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_15">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>Comentários</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QTableWidget" name="twgComentariosRA"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>370</y>
     <width>711</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_14">
    <item>
     <widget class="QLabel" name="label_13">
      <property name="text">
       <string>Adicionair comentário</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtComentario"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="btnRemoverRA">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>10</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="Remover Radioamador"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="imgs/imagens.qrc">
     <normaloff>:/PessoaRemover.png</normaloff>:/PessoaRemover.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGravarAlteracoes">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="Salvar os Dados"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="imgs/imagens.qrc">
     <normaloff>:/salvar.png</normaloff>:/salvar.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnInserirNovoRA">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>10</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="Novo Radioamador"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="imgs/imagens.qrc">
     <normaloff>:/PessoaNovo.png</normaloff>:/PessoaNovo.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnLocalizaInsereRA">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>50</width>
     <height>50</height>
    </rect>
   </property>
   <property name="toolTip">
    <string extracomment="Localizar Indicativo"/>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="imgs/imagens.qrc">
     <normaloff>:/localizar.png</normaloff>:/localizar.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>130</y>
     <width>41</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>ID</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtId">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="font">
       <font>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>130</y>
     <width>121</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_16">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Indicativo</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtIndicativo">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>189</x>
     <y>130</y>
     <width>71</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label_4">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>DMRID</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtDmrid">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>130</y>
     <width>231</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_14">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Criado</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtCriado">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>130</y>
     <width>261</width>
     <height>48</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_12">
    <item>
     <widget class="QLabel" name="label_15">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Modificado</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtModificado">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>171</width>
     <height>53</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_13">
    <item>
     <widget class="QLabel" name="label_16">
      <property name="text">
       <string>Indicativo a ser Localizado</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="edtPesquisaIndicativo">
      <property name="font">
       <font>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>edtQra</tabstop>
  <tabstop>edtNome</tabstop>
  <tabstop>edtSobrenome</tabstop>
  <tabstop>edtCidade</tabstop>
  <tabstop>edtEstado</tabstop>
  <tabstop>edtPais</tabstop>
  <tabstop>edtCelular</tabstop>
  <tabstop>edtEmail</tabstop>
  <tabstop>twgComentariosRA</tabstop>
 </tabstops>
 <resources>
  <include location="imgs/imagens.qrc"/>
 </resources>
 <connections/>
</ui>
